src/main/webapp/pages/entry_page.jsx

```diff
@@ -1,14 +1,22 @@
-import React from 'react';
+import React, { Suspense } from 'react';
 import layout from '@splunk/react-page';
 import { BrowserRouter as Router } from 'react-router-dom';
 import { SplunkThemeProvider } from '@splunk/themes';
+import { WaitSpinnerWrapper } from '../components/table/CustomTableStyle';
 
 import { StyledContainer, ThemeProviderSettings } from './EntryPageStyle';
 import { PAGE_CONF, PAGE_INPUT } from '../constants/pages';
 import ConfigManager from '../util/configManager';
-import InputPage from './Input/InputPage';
-import ConfigurationPage from './Configuration/ConfigurationPage';
 import messageDict from '../constants/messageDict';
+import { getBuildDirPath } from '../util/script';
+
+// eslint-disable-next-line no-undef,camelcase
+__webpack_public_path__ = `${getBuildDirPath()}/`;
```

Comment:
Can you verify if custom hooks, custom cell, etc. is loading fine?

Reply:
Custom Hook is working fine. and other custom features use similar type of import to load custom file.